src/api/ehr-out/send-core-message.js

```diff
@@ -4,7 +4,7 @@ import { logError } from '../../middleware/logging';
 import { updateExtractForSending } from '../../services/parser/message/update-extract-for-sending';
 import { initializeConfig } from '../../config/index';
 import { sendMessage } from '../../services/mhs/mhs-outbound-client';
-import { removeTitleFromExternalAttachments } from '../../services/mhs/mhs-attachments-wrangler';
+import {removeTitleFromExternalAttachments, wrangleAttachments} from '../../services/mhs/mhs-attachments-wrangler';
```

Comment:
It is best-practice to have spaces between the curly brackets and function names you are importing.